common/views/components/ArchiveTree/ArchiveTree.js

```diff
@@ -540,97 +502,103 @@ const ListItem = ({
           currentId: item.work.id,
           tree: fullTree,
         });
-        if (item.children) {
-          switch (event.key) {
-            case 'ArrowRight': {
-              // When focus is on an open node, moves focus to the first child node.
-              if (item.openStatus) {
-                if (nextId) {
-                  setTabbableId(nextId);
-                }
-              }
-
-              // When focus is on an end node, does nothing.
-              if (item.children && item.children.length === 0) {
-                return;
+        // if (item.children) {
+        switch (event.key) {
```

Comment:
Does IE 11 handle `event.key` consistently? I'm consistently frustrated that `keyCode` is deprecated but apparently the most consistently implemented. [I've been looking at doing something similar recently](https://github.com/wellcomecollection/wellcomecollection.org/pull/5614/files#diff-77dad37477d02a6b483961b2541ef8ff7b0e727fcd9d33671d7c0c6e173763f8R64-R103)

Reply:
ah, thanks - I'll nick that